troi/playlist.py

```diff
@@ -1,19 +1,33 @@
-import ujson
+import json
 import requests
 
-from troi import Recording, Playlist, PipelineError, Element 
+from troi import Recording, Playlist, PipelineError, Element
 from troi.operations import is_homogeneous
 
-LISTENBRAINZ_SERVER_URL = "https://test.listenbrainz.org"
+LISTENBRAINZ_SERVER_URL = "https://api.listenbrainz.org"
 LISTENBRAINZ_PLAYLIST_CREATE_URL = LISTENBRAINZ_SERVER_URL + "/1/playlist/create"
+PLAYLIST_TRACK_URI_PREFIX = "https://musicbrainz.org/recording/"
+PLAYLIST_ARTIST_URI_PREFIX = "https://musicbrainz.org/artist/"
+PLAYLIST_RELEASE_URI_PREFIX = "https://musicbrainz.org/release/"
+PLAYLIST_URI_PREFIX = "https://listenbrainz.org/playlist/"
+PLAYLIST_EXTENSION_URI = "https://musicbrainz.org/doc/jspf#playlist"
 
 def _serialize_to_jspf(playlist, created_for=None):
 
-    data = { "creator": "ListenBrainz Troi" }
+    data = { "creator": "ListenBrainz Troi",
+             "extension": {
+                 PLAYLIST_EXTENSION_URI: {
+                     "public": True
+                 }
+           }
+    }
 
     if playlist.name:
         data["title"] = playlist.name
 
+    if playlist.description:
+        data["annotation"] = playlist.description
```

Comment:
We should probably add this fields to docs at https://musicbrainz.org#jspf.

Reply:
annotation is a field defined in jspf itself.